webrtc.html

```diff
@@ -1873,28 +1800,100 @@ <h4>Set the RTCSessionDescription</h4>
+                            back, disassociate it and set <var>transceiver</var>'s
+                            <code><a data-link-for="RTCRtpTransceiver">mid</a></code> value
+                            to <code>null</code>.</p>
+                          </li>
+                          <li>
+                            <p>Set <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\SenderTransport]]</a>
+                            to <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\StableStateSenderTransport]]</a>
+                            and set <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\StableStateSenderTransport]]</a>
+                            to <code>null</code>.</p>
+                          </li>
+                          <li>
+                            <p>Set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\ReceiverTransport]]</a>
+                            to <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\StableStateReceiverTransport]]</a>
+                            and set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\StableStateReceiverTransport]]</a>
+                            to <code>null</code>.</p>
+                          </li>
+                          <li>
+                            <p>Set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\ReceiveCodecs]]</a>
+                            to <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\StableStateReceiveCodecs]]</a>
+                            and set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\StableStateReceiveCodecs]]</a>
+                            to an empty list.</p>
+                          </li>
+                          <li data-tests="RTCPeerConnection-setDescription-transceiver.html">
+                            <p>If the <var>transceiver</var> was created by applying the
+                            <code><a>RTCSessionDescription</a></code> that is
+                            being rolled back, and a track has never been attached to
+                            it via <code>addTrack</code>, run the following steps:</p>
+                            <ol>
+                              <li>
+                                <p><a data-lt="stop the RTCRtpTransceiver">Stop the
+                                RTCRtpTransceiver</a> <var>transceiver</var>.</p>
+                              </li>
+                              <li>
+                                <p>Remove <var>transceiver</var> from <var>connection</var>'s
+                                <a href="#transceivers-set">set of transceivers</a>.</p>
+                              </li>
+                              <li>
+                                <p><a>Set the associated remote streams</a> given
+                                <var>transceiver</var>.<a>[[\Receiver]]</a>, <var>msids</var>
+                                as the empty list, <var>addList</var>, and <var>removeList</var>.</p>
```

Comment:
I'd go with "an empty list", however now that I introduced a [[LastStableStateAssociatedRemoteMediaStreams]] internal slot, I perform this step above instead of handling removal and non-removal differently when it comes to resetting the streams.